tkg/fakes/config/bom/tkg-bom-v1.3.1.yaml

```diff
@@ -89,6 +89,12 @@ components:
       capzControllerImage:
         imagePath: cluster-api/cluster-api-azure-controller
         tag: v0.4.8-47-gfbb2d55b_vmware.1
+  cluster-api-ipam-provider-in-cluster:
```

Comment:
This section is actually in the file already via https://github.com/vmware-tanzu/tanzu-framework/pull/4529 a couple of lines up. So you should be able to remove this section.